core/src/main/java/google/registry/model/tld/Tld.java

```diff
@@ -416,34 +417,42 @@ public ImmutableSet<String> getReservedListNames() {
    * amount of time following creation.
    */
   @Column(nullable = false)
+  @JsonSerialize(using = DurationSerializer.class)
```

Comment:
And there's no way to apply this by default to all `Duration`s?  Having to include it individually for each field seems like it might be prone to forgetting to do it.  There's no `Duration` we wouldn't want to serialize in this way, right?